.github/workflows/ci.yml

```diff
@@ -51,7 +51,7 @@ jobs:
     strategy:
       matrix:
         workspace: ${{ fromJSON(needs.find-changed-workspaces.outputs.workspaces) }}
-        node-version: [18.x, 20.x]
+        node-version: [20.x, 22.x]
```

Comment:
Why remove 18.x?

Reply:
Only in the CI matrix, Node.js 20 and 22 are the only supported versions supported by the Backstage framework and we agreed it made sense to align with the versions they test -https://github.com/backstage/backstage/blob/92013bfee9d5d788a75a44a7a7d3916964f113b7/.github/workflows/ci.yml#L22

Note this doesn't impact what workspaces/plugins can claim support for - just what is tested.

Reply:
Official Backstage Node policy for reference: https://backstage.io/docs/overview/versioning-policy#nodejs-releases